shared/lib/state_manager/state_manager.js

```diff
@@ -99,9 +117,16 @@ export default class StateManager {
   getInitialData(){
     let state_manager = this;
     // we'll load past user answers and get CC results here.
+    state_manager.checkLocalStorage();
     return state_manager.updateDefaults();
   }
 
+  checkLocalStorage(){
+    let state_manager = this;
+    if (state_manager.average_footprint_storage) Object.assign(state_manager.state.average_footprint, state_manager.average_footprint_storage)
+    if (state_manager.user_footprint_storage) Object.assign(state_manager.state.user_footprint, state_manager.user_footprint_storage)
+  }
+
```

Comment:
Take action values are now also stored in localStorage:
```javascript
  updateTakeActionResultStorage() {
    let state_manager = this;
    localStorage.setItem('result_takeaction_dollars', JSON.stringify(state_manager.state['result_takeaction_dollars']));
    localStorage.setItem('result_takeaction_net10yr', JSON.stringify(state_manager.state['result_takeaction_net10yr']));
    localStorage.setItem('result_takeaction_pounds', JSON.stringify(state_manager.state['result_takeaction_pounds']));
  }
```
These get updated by `parseTakeactionResult`